### PR TITLE
Added to README file

### PR DESCRIPTION
**Pull request by jocelynshen**
Added links to README file to make our github page easier to navigate

### CODE REVIEW

**Comment by nzufelt** [comment]
Can we perhaps have a discussion about what should go into the readme?  I'm not sure that links to some subset of the resources is what we want there.  The readme should be a landing page for those interested in contributing.  So to start: what is this repository, how might they use it (not totally applicable here), and how might they contribute?  What are we looking for with contributions?

File: README.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 
 This repo holds all the publically available content for the Honors Computer Science Seminar at Phillips Academy Andover, CSC630: The Open Source Movement.
 
-## Contributing
+[Contribution guidelines](CONTRIBUTING.md)
+<br/>[Code of conduct](CODE_OF_CONDUCT.md)
```
Comment (ruddfawcett) [inline README.md:6]: Can you edit this so it doesn't have a `<br/>`?

File: README.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
 
-Contributions to the website are both welcomed and appreciated. See the [CONTRIBUTING.md](https://github.com/nzufelt/open_source_movement_csc630/blob/master/CONTRIBUTING.md) file to read more about how to contribute.
+[Resources:](resources/resources.md)
+* [Learning git](resources/learning_git.md)
+* [Connecting to Github via SSH](resources/connecting_to_github_via_ssh.md)
 
-## Link to website
-
-Visit the website at https://nzufelt.github.io/open_source_movement_csc630/.
+[Assignments:](assignments)
```
Comment (ruddfawcett) [inline README.md:12]: Also, maybe can we link these back tot he assignments page on the site?

**Comment by ruddfawcett** [comment]
Hey @jocelynshen, just looking at this now — added some comments — let me know any thoughts you may have :)

**Comment by ruddfawcett** [comment]
Hey, @jocelynshen, any thoughts on the above?

**Pull request closed without merging**